cert-manager/cluster/MutatingWebhookConfiguration-cert-manager-webhook.yaml

```diff
@@ -1,33 +1,35 @@
-# Source: cert-manager/templates/webhook-mutating-webhook.yaml
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
-  name: cert-manager-webhook
+  annotations:
+    cert-manager.io/inject-ca-from-secret: cert-manager/cert-manager-webhook-ca
   labels:
     app: webhook
-    app.kubernetes.io/name: webhook
+    app.kubernetes.io/component: webhook
     app.kubernetes.io/instance: cert-manager
-    app.kubernetes.io/managed-by: Tiller
-    helm.sh/chart: cert-manager-v0.13.1
-  annotations:
-    cert-manager.io/inject-ca-from-secret: "cert-manager/cert-manager-webhook-tls"
+    app.kubernetes.io/name: webhook
+  name: cert-manager-webhook
 webhooks:
-  - name: webhook.cert-manager.io
-    rules:
-      - apiGroups:
-          - "cert-manager.io"
-          - "acme.cert-manager.io"
-        apiVersions:
-          - v1alpha2
-        operations:
-          - CREATE
-          - UPDATE
-        resources:
-          - "*/*"
-    failurePolicy: Fail
-    sideEffects: None
-    clientConfig:
-      service:
-        name: cert-manager-webhook
-        namespace: "cert-manager"
-        path: /mutate
+- admissionReviewVersions:
+  - v1
+  - v1beta1
```

Comment:
We have apiVersions: v1alpha2 on the old version. My guess is as you spoke, the Mutation webhook deals with conversions, right?

Reply:
This is the list of *AdmissionReview* versions that the webhook supports, rather than the versions of the resources being mutated that should be submitted to the webhook :)

Reply:
Thanks!